hal/halx86/smp/amd64/apentry.S

```diff
@@ -0,0 +1,43 @@
+/*
+ * PROJECT:     ReactOS Kernel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     AMD64 Application Processor (AP) spinup setup
+ * COPYRIGHT:   Copyright 2021 Justin Miller <justinmiller100@gmail.com>
```

Comment:
```suggestion
 * COPYRIGHT:   Copyright 2023 Justin Miller <justinmiller100@gmail.com>
```
same here. We added it 2023. I will not point to more hits, although some other newly added files might be affected as well.